### ISSUE TITLE
Non-ASCII quote marks.

### ISSUE THREAD

**Issue by lfaller** [conchoecia/pauvre#30]
https://github.com/conchoecia/pauvre/blob/cc3555c2dac598e51edf96c047ab8d62d8e5c2bb/pauvre/marginplot.py#L50

I am running into the following issue with your code:

```
SyntaxError: Non-ASCII character '\xe2' in file /usr/local/lib/python2.7/dist-packages/pauvre/marginplot.py on line 50, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

I believe this is due to fancy unicode quote marks.

Thanks!

**Comment by wdecoster** [comment]
Hi lfaller,

Thanks for reporting this. Those quote marks are in a doc string, essentially a non-coding part of the script. So changing this wouldn't be a problem, I'll open a pull request later today.

However, I would advise you (if your system permits) to switch to Python3. Python2 will be around for two more years and then it goes (hopefully) to the dustbin of history. I understand that for some legacy tools/setups it's hard or impossible to change, but for the rest, there is as far as I know no real excuse.

Cheers,
Wouter

**Comment by wdecoster** [comment]
In addition to my previous reaction: no, it turns out pauvre is not compatible with Python 2.7. This also is specified in the README: https://github.com/conchoecia/pauvre#requirements

**Comment by conchoecia** [comment]
I'm not sure how to proceed since this is an error with python 2 as @wdecoster described, but I will add some additional encoding information at the top of each file to try to prevent this problem in the future. This probably stems from me using a non-US keyboard.

I added `# -*- coding: utf-8 -*-` to the top of every file to attempt to fix the problem in commit b8053815a06b2ed5234e9fb4a04eb629bf71ed4e . This problem is discussed in [this SO thread](https://stackoverflow.com/questions/21639275/python-syntaxerror-non-ascii-character-xe2-in-file). @lfaller, Please reopen this problem if it still doesn't work and I will try to find the extra byte that python2.7 doesn't like.
